plugin/trino-singlestore/pom.xml

```diff
@@ -29,7 +29,7 @@
         <dependency>
             <groupId>com.singlestore</groupId>
             <artifactId>singlestore-jdbc-client</artifactId>
-            <version>1.2.8</version>
+            <version>1.2.9</version>
```

Comment:
this commit is "dependabot[bot] authored and ebyhr committed 9 hours ago"

since it's not just a version bump, make this commit yours.
when someone looks at git blame of a java class, seeing "dependabot" as code author doesn't help at all